contracts/swap/factory.go

```diff
@@ -19,7 +19,7 @@ var (
 	// Deployments maps from network ids to deployed contract factories
 	Deployments = map[uint64]common.Address{
 		// Ropsten
-		3: common.HexToAddress("0x2e9C43E186eaF4fee10799d67e75f8CFc5BA3a0c"),
```

Comment:
TODO here, because there are problems with auto-generating the go-bindings @ralph-pichler

Reply:
This causes tests to fail. You might wanna enter some address as a placeholder.

Reply:
Done